hadoop-hdds/docs/content/feature/OM-HA.zh.md

```diff
@@ -58,7 +58,7 @@ Ozone Manager 和 Storage Container Manager 都支持 HA。在这种模式下，
  ```
```

Comment:
Same here:

```suggestion
```XML
```

Reply:
@adoroszlai , sorry,  didn't refresh the page and notice your comments.

Reply:
resolved

Reply:
Thanks @ChenSammi @adoroszlai for reviewing . I will open another PR for XML formatting.